configs/constants.js

```diff
@@ -6,8 +6,16 @@ const SRC_DIR = path.resolve(ROOT_DIR, 'webapp');
 const SERVER_DIR = path.resolve(ROOT_DIR, 'server');
 const MODULES_DIR = path.resolve(ROOT_DIR, 'node_modules');
 
+const resolveRoot = function(module) {
```

Comment:
Can we put this in a `configs/utils` file? It doesn't feel like it belongs with constants.